docs/format_signatures.rst

```diff
@@ -0,0 +1,122 @@
+Formatting signatures
+=====================
+
+This theme provides two alternative ways to automatically format/indent API
+documentation signatures.
+
+CSS-based function parameter wrapping
+-------------------------------------
+
+There is a CSS-based formatting rule that can be enabled for long function
+signatures that displays each function parameter on a separate line.  This is
+enabled by default, and works fairly well for Python signatures.
+
+.. confval:: html_wrap_signatures_with_css
+
+   Specifies for which `Sphinx domains
+   <https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html>`__
+   this CSS-based formatting is enabled.
+
+   Supported types are:
+
+   :py:obj:`bool`
+       If `True`, enable CSS-based formatting for all domains.  If `False`,
+       disable CSS-based formatting for all domains.
+
+   :py:obj:`list[str]`
+       List of domains for which to enable CSS-based formatting.
+
+   The default value is :python:`True`
+
+.. confval:: html_wrap_signatures_with_css_column_limit
+
+   Maximum signature length before function parameters are displayed on separate
+   lines.
+
+   Only applies to domains for which :confval:`html_wrap_signatures_with_css` is
+   enabled.
+
+   The default value is :python:`70`.
+
+.. rst-example::
+
+   .. py:function:: long_function_signature_example(\
+                      name: int, \
+                      other_param: list[str], \
+                      yet_another: bool = False, \
+                      finally_another: str = "some long string goes here"\
+                    ) -> Tuple[A, Very, Long, Return, Type]
```

Comment:
I'm the RTD CI, you'll see that this return type is styled like a hyperlink, but it doesn't actually behave like a hyperlink (only the color changes when clicking it - nothing else happens).

I would expect that `Tuple` and `Type` (a built-in python function) lead to the python docs, but it doesn't.

Reply:
I fixed the hover effect per your suggestion, and also fixed the issue with the close paren not being on a separate line --- `:last-of-type` matches based on the tag name, not the class.

I think fixing the cross-linking of Tuple will require pulling in some other changes I have in the tensorstore docs.